Gems/Atom/Tools/AtomToolsFramework/Code/Source/Util/Util.cpp

```diff
@@ -449,11 +451,17 @@ namespace AtomToolsFramework
                 {
                     if (ValidateDocumentPath(path))
                     {
-                        results.insert(path);
+                        results.push_back(path);
                     }
                 }
             }
         }
+
+        // Sorting the container and removing duplicate paths. This isn't entirely necessary but the function previously returned a sorted
```

Comment:
Should we just forego the sort? Is the reason not to just return an `unordered_set` instead that there's no mapping for that in python?

Reply:
Sorry, I should have commented on that. Yes, Python complained about set<string> so I changed it to a compatible type. I could have written a Python specific adapter function but did not see the point. I could remove the sort/unique from this function and move it to the call site.

Reply:
On the other hand, I think it's possible to have nested scan folders. That might result in the same file getting added to the vector multiple times. In order to remove duplicate entries, I have to sort before calling unique. Alternatively, I could have the function build a set or unordered set and return a vector constructed from the set.